README.md

```diff
@@ -4,33 +4,40 @@ Open source voice assistant development. Make your own minds!
 
 Open Assistant is a private open source personal assistant prototype able to complete operating system tasks using vocal commands.
 
-# Installation
+## Installation
 
-## Windows
+### Windows
 
-Windows (Click & Run)
+#### _Click & Run_
 
-* Download http://openassistant.org/download/oa_0.21_windows.zip
+* Download [http://openassistant.org/download/oa_0.21_windows.zip](http://openassistant.org/download/oa_0.21_windows.zip)
 * Unzip package
 * Launch `oa.exe` in "Open Assistant" folder
 
-Windows Python Install (Python 3.x)
+#### _Python 3_
 
 * Install [common list of Python packages](https://www.python.org/downloads/windows/)
 * Install Windows-specific dependencies: ``pip install pywin32``
 * Install Python dependencies: `pip install -r requirements.txt`
 
-## Ubuntu Linux
+### Mac OS X
+
+* Install [brew](https://brew.sh/) if you have not already.
+* Install pocketsphinx dependencies: `brew install swig`, `brew install espeak`, and `brew install openal-soft`
```

Comment:
* `espeak` isn't currently used on macOS -- maybe that'll change
* single command

```suggestion
* Install pocketsphinx dependencies: `brew install swig openal-soft`
```